canvas_modules/common-canvas/src/notification-panel/notification-panel.scss

```diff
@@ -16,6 +16,8 @@
 
 $notification-panel-arrow-height: 8px;
 $notification-panel-header-height: 35px;
+$notification-panel-clear-all-height: 49px;
+$notification-message-height: 64px;
```

Comment:
This file has a strange mixture of the two scss variables above plus some 'hardcoded' pixel dimenions in the rulesets below (like notification-panel width).  Can we move any 'hardcoded' sizes into this set of variables?

Reply:
`$notification-message-height` is used in several places and `$notification-panel-clear-all-height` is used in a calculation. the width was only used in once place, as you said, so that's the only new spacing change that I did not create a variable for